server/src/testflinger/api/v1.py

```diff
@@ -784,8 +784,17 @@ def check_valid_uuid(job_id):
 
 
 @v1.get("/job/<job_id>/position")
-def job_position_get(job_id):
-    """Return the position of the specified jobid in the queue."""
+# TODO: This is billed as providing the position in a queue describing at what
+#  point the job will be picked up, however there are certainly cases where
+#  this is an oversimplification as each agent listens to multiple queues.
+#  Therefore, there could be countless other jobs keeping the agents for the
+#  queue which this job is in busy rather than servicing this job.
+def job_position_get(job_id: str) -> tuple[str, int]:
```

Comment:
This optionally also return just a string
```suggestion
def job_position_get(job_id: str) -> str | tuple[str, int]:
```

and I just noticed it also returns a dict! 
```suggestion
def job_position_get(job_id: str) -> dict | str | tuple[str, int]:
```

We don´t have type checkers but maybe for accuracy use option 2?